executor/containerdexecutor/executor.go

```diff
@@ -100,6 +104,15 @@ func (w containerdExecutor) Exec(ctx context.Context, meta executor.Meta, root c
 	if system.SeccompSupported() {
 		opts = append(opts, seccomp.WithDefaultProfile())
 	}
+	if w.cgroupParent != "" {
+		var cgroupsPath string
+		if strings.Contains(w.cgroupParent, ":") {
```

Comment:
shouldn't we detect ".slice" as well? `:` is not a forbidden path char

Reply:
right we should check for that.